.eleventy.js

```diff
@@ -90,20 +79,28 @@ module.exports = function (eleventyConfig) {
   eleventyConfig.addShortcode("version", () => config.version)
   eleventyConfig.addFilter("limit", (array, limit) => array.slice(0, limit))
   eleventyConfig.addFilter("timeToRead", getReadTime)
-  eleventyConfig.addFilter("postDate", dateString =>
-    DateTime.fromISO(dateString).toLocaleString(DateTime.DATE_MED)
-  )
+  eleventyConfig.addFilter("postDate", date => {
+    if (date && typeof date.getMonth === "function") {
+      return DateTime.fromJSDate(date).toLocaleString(DateTime.DATE_MED)
+    }
+    if (typeof date === "object") {
+      return DateTime.fromObject(date).toLocaleString(DateTime.DATE_MED)
+    }
+    return DateTime.fromISO(date).toLocaleString(DateTime.DATE_MED)
```

Comment:
TODO: update blog post date formats to be consistent (remove quotes in frontmatter)